src/main/java/net/sf/jabref/gui/importer/fetcher/GoogleScholarFetcher.java

```diff
@@ -193,13 +165,21 @@ private static void runConfig() throws IOException {
     }
 
     private String getCitationsFromUrl(String urlQuery, Map<String, JLabel> ids) throws IOException {
-        String cont = new URLDownload(urlQuery).downloadToString(StandardCharsets.UTF_8);
-        Matcher m = GoogleScholarFetcher.BIBTEX_LINK_PATTERN.matcher(cont);
+        String cont = URLDownload.createURLDownloadWithBrowserUserAgent(urlQuery).downloadToString(StandardCharsets.UTF_8);
+
+        Matcher m = GoogleScholarFetcher.SCHOLAR_ID_PATTERN.matcher(cont);
         int lastRegionStart = 0;
 
         while (m.find()) {
-            String link = m.group(1).replace("&amp;", "&");
-            link = link+"&oe=utf-8"; // append param 'oe=utf-8' to tell google to serve UTF-8 encoded results
+
+            String citationsPageURL = CITATIONS_PAGE_URL_BASE+m.group(1)+CITATIONS_PAGE_URL_SUFFIX;
```

Comment:
Mhmm. I don't think this is that much an improvement as still some string concatenation is needed for the param "q" with `"info:"+id+":scholar.google.com/"`